src/fft_small.h

```diff
@@ -170,145 +173,75 @@ do { \
 } while (0)
 
 
-#define SD_FFT_CTX_INIT_DEPTH 12
+#define SD_FFT_CTX_W2TAB_INIT 12
+#define SD_FFT_CTX_W2TAB_SIZE 40
 
 /* This context is the one expected to sit in a global position */
 typedef struct {
     double p;
     double pinv;
     nmod_t mod;
     ulong primitive_root;
-    ulong blk_sz;
     volatile ulong w2tab_depth;
```

Comment:
Already in the code base, but why is one volatile?

Reply:
The ill-defined volatile keyword was the best one could do with plain ansi-C. I see now that declarations can be mixed with code and c++ style comments are allowed. Has flint moved into the 21st century now, i.e. can we use the atomics of C11?

Reply:
Yes! C11 atomics are already used in the fmpz memory manager.